CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## [v18.0.0] 2023-08-28
 
 ### Notable Changes
+- **CUMULUS-3392**
+- Modify cloudwatch rule by deleting `custom`
```

Comment:
Can you move this above to the `Unreleased` section and create a new section called `### Changed`?